python/pylibcudf/pylibcudf/column.pyx

```diff
@@ -333,6 +337,39 @@ cdef class Column:
         )
         cdef gpumemoryview data = gpumemoryview(rmm_buff)
 
+        if verify:
```

Comment:
@vyasr what would be the best way to test `verify=False` since this is a `cdef` method?